src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -81,6 +81,23 @@ foam.CLASS({
       name: 'browseSubtitle',
       factory: function() { return 'View all ' + this.browseTitle.toLowerCase() + '.' }
     },
+    {
+      class: 'FObjectProperty',
+      name: 'primaryAction',
+      documentation: `
+        The most important action on the page. The view for this controller may
+        choose to display this action prominently.
+      `,
+      value: null
+    },
+    {
+      class: 'Boolean',
+      name: 'disableBrowseViewDataSet',
```

Comment:
What does this do?

Reply:
Now that you bring this up, I think we don't need this.